packages/scene-composer/tests/components/three-fiber/StatsWindow.spec.tsx

```diff
@@ -1,29 +0,0 @@
-/* eslint-disable import/first */
```

Comment:
These weren't deleted so much as completely rewritten and moved to model our coding standards.